spec/fixtures/sample2.js

```diff
@@ -0,0 +1,27 @@
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية and some english
+لغة عربية لغة عربية لغة عربية لغة عربية
+
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
+لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية لغة عربية
```

Comment:
Adding some persian to make the test more compelte:
```suggestion
زبان پارسی  زبان پارسی زبان پارسی زبان پارسی زبان پارسی زبان پارسی زبان پارسی
زبان پارسی زبان پارسی زبان پارسی زبان پارسی زبان پارسی زبان پارسی
 پچژگ پچژگ پچژگ پچژگ پچژگ
```